webapp/packages/plugin-sql-editor/src/locales/en.ts

```diff
@@ -45,4 +45,8 @@ export default [
   ['plugin_sql_editor_settings_import_max_size_description', 'Maximum file size that can be imported in KB'],
   ['plugin_sql_editor_settings_auto_save', 'Auto save'],
   ['plugin_sql_editor_settings_auto_save_description', 'Automatically save the editor content'],
+  [
+    'plugin_sql_editor_settings_auto_save_description_git_integration',
+    'Automatically save the editor content. This feature is only available when Git integration is turned off',
```

Comment:
By default, SQL editor content is saved in local storage, but this feature kicks in when the editor is opened to edit an SQL script file.

This message may confuse users, as it controls only script-saving behaviour and doesn't affect saving in local storage